servers/physics_2d/joints_2d_sw.cpp

```diff
@@ -408,8 +408,7 @@ bool GrooveJoint2DSW::setup(real_t p_step) {
 	//joint->bias = cpvclamp(cpvmult(delta, -joint->constraint.biasCoef*dt_inv), joint->constraint.maxBias);
 
 	Vector2 delta = (B->get_transform().get_origin() + rB) - (A->get_transform().get_origin() + rA);
-	real_t _b = get_bias();
-	_b = 0.001;
+	real_t _b = 0.001;
```

Comment:
This appears to be like this since Godot was open sourced (first commit).

I suggest you change that code to:
```
// FIXME: We used to do this assignment and then override it with 0.001 right after,
// investigate why.
//real_t _b = get_bias();
real_t _b = 0.001;
```